app/views/posts/index.html.erb

```diff
@@ -1,2 +1,5 @@
-<h1>Posts of a given user</h1>
-<p>All of the posts made by the given user get listed here</p>
+<%= render partial: 'users/user_card', locals: {user: @posts.first.user} %>
+
+<% @posts.each do |post| %>
+  <%= render partial: 'post_card', locals: { post: post, comments: post.recent_five_comments} %>
+<% end %>
```

Comment:
- [ ] I have created many posts in the rails console however when I click on the see all posts button I see this error, please fix it.

![image](https://user-images.githubusercontent.com/55635977/172184848-65833698-dde5-43ef-b57f-d53c6e6ccd9b.png)

Reply:
Thanks for your review.
May I know how you made posts? Because I already have multiple users and posts without any issue.

Reply:
@Emmyn5600 Thanks for your review.
May I know how you made posts? Because I already have multiple users and posts without any issue.

Reply:
I guess you have used `Post.create(author: user1, ...`. I don't have `author`. I have `user` in other tables.

Reply:
In terminal:
![post-first-user](https://user-images.githubusercontent.com/11241315/172196223-2e076bc0-b223-476e-8d9d-82e8df395e5d.png)

In post view:
![image](https://user-images.githubusercontent.com/11241315/172196521-f93a967f-a4a6-4d8a-a71d-40592b9cc556.png)

Reply:
The same page as you posted. I see 

![image](https://user-images.githubusercontent.com/11241315/172198261-9be5bb35-9414-49f5-a6f7-9ddb725a6c0f.png)

![image](https://user-images.githubusercontent.com/11241315/172198157-371e0364-138a-40ae-bae5-f17ff28e46e4.png)

Reply:
Ok!

The issue is about those users with no posts. I forgot to handle such cases.

Reply:
Sure @ahangarha , I enjoyed the call and our conversation too. Please implement the changes and request for another review

Reply:
I did